tests/e2e/data/config-flextesa-updated-sandbox.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Is this file used anywhere in this PR?

Reply:
Not yet, but it is part of the test that has an issue now.
I have added an issue in the description.

Reply:
I added link to the issue in the description.